templates/injector-deployment.yaml

```diff
@@ -118,6 +118,8 @@ spec:
             - name: AGENT_INJECT_TEMPLATE_STATIC_SECRET_RENDER_INTERVAL
               value: "{{ .Values.injector.agentDefaults.templateConfig.staticSecretRenderInterval }}"
             {{- end }}
+            - name: AGENT_INJECT_VAULT_NAMESPACE
+              value: "{{ .Values.injector.agentDefaults.namespace }}"
```

Comment:
I think this var should only be set if `.Values.injector.agentDefaults.namespace` isn't empty, similar to how `AGENT_INJECT_TEMPLATE_STATIC_SECRET_RENDER_INTERVAL` is handled above.